src/presenter/board-presenter.js

```diff
@@ -135,8 +149,8 @@ export default class BoardPresenter {
       case UserAction.UPDATE_FILM:
         try {
           await this.#filmModel.updateFilm(updateType, update);
-        } catch {
-          console.log('Can\'t update film');
+        } catch (error) {
+          throw error.message;
```

Comment:
javascript позволяет бросать в качестве исключения все подряд...
но лучше все таки экзеплятры классов-наследников Error
Если ты хотел сделать здесь место для "посмотреть" с брейкпоинтом в дебагере,
то 
```suggestion
          throw error;
```
Если же ты хотел спрятать стэк-трейс, то 

```suggestion
          throw new Error(error.message);
```